### PR TITLE
chore(monorepo): consolidate NODE_ENV values

### LINKED ISSUE #822: NODE_ENV is a mess
Some of our projects use `dev` and `prod` for `NODE_ENV`, others use `development` and `production`. Then the email service was made to use a hybrid `dev`/`production` mix for it's env var, and I've now duplicated that into the event broker when I copied a bunch of code across in #794.

None of that's doing any harm of course, but it's one more paper-cut in the spirit of #808. Do we care enough to harmonise them across projects? I'm not sure if the changes would be entirely limited to this repo or if we'd need to change `cloudops-deployment` too, which might take this into too-messy-to-bother-fixing territory.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-806)
┆Issue Number: FXA-806

### PR DESCRIPTION
**Pull request by meandavejustice**
- fixes #822

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -103,8 +103,8 @@ jobs:
       - image: circleci/mysql:5.7.27
       - image: mozilla/fxa-email-service
         environment:
-          NODE_ENV: dev
-          FXA_EMAIL_ENV: dev
+          NODE_ENV: development
+          FXA_EMAIL_ENV: development
```
Comment (meandavejustice) [inline .circleci/config.yml]: Need to get the 👍 from someone more familiar with our email server than myself

**Comment by vladikoff** [comment]
auth test fail in:

```
  360 passing (1m)
  1 failing

  1) /v1
       "after all" hook:
     TypeError: subhub.close is not a function
      at Object.close (bin/key_server.js:135:20)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

**Comment by lmorchard** [comment]
@meandavejustice Yeah, any changes to env vars need to be added to the deploy doc for this sprint so ops is aware updates that need doing. I added a note for config changes, but may need to verify that this PR is merged before the deploy to make sure it's the right time to make the change

File: packages/fxa-auth-db-mysql/README.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ where `NODE_ENV` is an environment variable
 set in the shell.
 
 For local development,
-set `NODE_ENV` to `dev`
+set `NODE_ENV` to `development`
 then create a new JSON file
 called `config/dev.json`.
```
Comment (jrgm) [inline packages/fxa-auth-db-mysql/README.md]: ... called `config/development.json`

**Comment by bbangert** [comment]
@meandavejustice Does this PR still apply? Can it be updated and merged?

**Comment by meandavejustice** [comment]
@bbangert Rebased yesterday, were still having some test failures. This pr is just a nit/cleanup task so if we need to clear the queue I'm happy to have it closed. Otherwise I'll just continue to poke at it when I have free time

**Pull request closed without merging**